bookdown/03-pipelines/03-pipelines.Rmd

```diff
@@ -1,5 +1,580 @@
 # Pipelines {#pipelines}
 
-```{block, type='warning'}
-Work in progress :)
+```{r 03-pipelines-001_setup, message = FALSE, warning = FALSE, echo = FALSE, include = FALSE}
+knitr::opts_knit$set(root.dir = fs::dir_create(tempfile()))
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+library(visNetwork)
+library(mlr3pipelines)
```

Comment:
I don't think we have any setup chunks for specific chapters anymore. If you need to load libraries, just do it user-visible in the place where its needed.